src/main/resources/doc/en/html/libs/gates/buffer.html

```diff
@@ -173,7 +173,7 @@ <h2>
           East edge :
         </dt>
         <dd>
-          The output, which always matches the input into the left side. <b class="propertie">Bits attribute</b>.
+          The output, which always matches the input into the left side. Bit width according to Data  <b class="propertie">Bits attribute</b>.

```

Comment:
That's bit odd looking. Shouldn't the "Data Bits" be of class "properties", not just "Bits"?